src/session/basic_group.rs

```diff
@@ -109,4 +127,73 @@ impl BasicGroup {
         self.imp().member_count.set(member_count);
         self.notify("member-count");
     }
+
+    pub(crate) fn status(&self) -> BoxedChatMemberStatus {
+        self.imp().status.borrow().as_ref().unwrap().to_owned()
+    }
+
+    pub(crate) fn set_status(&self, status: BoxedChatMemberStatus) {
+        if self.imp().status.borrow().as_ref() == Some(&status) {
```

Comment:
```suggestion
        if self.status() == status {
```

Reply:
Hmm. I just saw that my change would crash, because you unwrap in `status()`. So either return an `Option` in `status` or ignore this. Depends on what @melix99 prefers.

Reply:
As I wrote in the other review, this is fine for now.